back/app/models/basket.rb

```diff
@@ -119,7 +119,12 @@ def update_basket_and_vote_counts(count_phases, update_context)
   def basket_submission
     return unless submitted?
 
-    if phase.voting_min_total && (total_votes < phase.voting_min_total)
+    if phase.voting_min_selected_options.to_i.positive? && (baskets_ideas.size < phase.voting_min_selected_options)
```

Comment:
```suggestion
    if baskets_ideas.size < phase.voting_min_selected_options
```

If the column cannot be `nil`, you can just do this (see migration)